recipes/openjph/build.sh

```diff
@@ -0,0 +1,14 @@
+set -ex
+
+mkdir -p build
+pushd build
+
+# OJPH_BUILD_EXECUTABLES would require libtiff which creates a circular
```

Comment:
This comment is outdated? :)

Reply:
yes, though i'm still curious about how to break that circular dependency in the future. these binaries leave alot to be desired.